Makefile

```diff
@@ -318,6 +320,11 @@ PORTLAYER_DEPS ?= lib/apiservers/portlayer/swagger.json \
 				  lib/apiservers/portlayer/restapi/options/*.go \
 				  lib/apiservers/portlayer/restapi/handlers/*.go
 
+$(admiralapi-client): lib/config/dynamic/admiral/swagger.json $(SWAGGER)
```

Comment:
This seems like the sort of thing that should be checked out/pulled from admiral's repo